frontend/appflowy_flutter/lib/mobile/presentation/home/turn_box.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/widgets.dart';
+
+class TurnBox extends StatefulWidget {
```

Comment:
AnimatedRotation is just an implicit version of using `RotationTransition`, which means we can avoid having this widget, what do you think?